scripts/notify-slack

```diff
@@ -0,0 +1,142 @@
+#!/usr/bin/env ruby
```

Comment:
This is imported from [identity-devops](https://gitlab.login.gov/lg/identity-devops/-/blob/main/kitchen/cookbooks/login_dot_gov/templates/default/notify-slack?ref_type=heads) where it's lived for a while as part of our infra notifications, I'll add some specs here